queries.sql

```diff
@@ -0,0 +1,9 @@
+/*Queries that provide answers to the questions from all projects.*/
+
+SELECT * from animals WHERE name LIKE '%mon';
+SELECT name from animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT name from animals WHERE neutered = 'true' AND escape_attempts < 3;
+SELECT date_of_birth from animals WHERE name = 'Agumon' OR name = 'Pikachu';
+SELECT name and escape_attempts from animals WHERE weight_kg > 10.5;
+SELECT * from animals WHERE neutered IS 'true';
+SELECT * from animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
```

Comment:
- [ ] You should add a query to Find all animals not named Gabumon, kindly make sure to implement this so all the quests are included

Reply:
Thank you @Ismailco , I will correct this!